x-pack/plugins/infra/common/http_api/metrics_explorer_views/v1/get_metrics_explorer_view.ts

```diff
@@ -6,7 +6,10 @@
  */
 
 import * as rt from 'io-ts';
+import { metricsExplorerViewRT } from '../../../metrics_explorer_views';
 
 export const getMetricsExplorerViewRequestParamsRT = rt.type({
   metricsExplorerViewId: rt.string,
 });
+
+export type GetMetricsExplorerViewResposePayload = rt.TypeOf<typeof metricsExplorerViewRT>;
```

Comment:
```suggestion
export type GetMetricsExplorerViewResposePayload = rt.TypeOf<typeof metricsExplorerViewRT>;
```

typo missing 'n' in GetMetricsExplorerViewResponsePayload

Reply:
🤦‍♂️